packages/api/src/controllers/session.ts

```diff
@@ -40,9 +45,25 @@ const fieldsMap = {
   recordingStatus: `session.data->'recordingStatus'`,
 };
 
+function toStringValues(obj: Record<string, any>) {
+  const strObj: Record<string, string> = {};
+  for (const [key, value] of Object.entries(obj)) {
+    strObj[key] = value.toString();
+  }
+  return strObj;
+}
+
 app.get("/", authMiddleware({}), async (req, res, next) => {
-  let { limit, cursor, all, order, filters, userId, parentId, count } =
-    req.query;
+  let {
```

Comment:
This just for typescript purposes? What would these things come in as if not for strings?

Reply:
This is actually not only a type issue, but the typing exposes a possible issue in code that only uses a `req.query.param` assuming it is always a `string`. The newer `@types/express` have a proper type for the `req.query` property which exposes the issue:
```
interface ParsedQs { [key: string]: undefined | string | string[] | ParsedQs | ParsedQs[] }
```

Which basically means that a `req.query` can contain a string, an array of strings or even recursive `ParsedQs` objects or array of those objects. That can happen on these cases:
 - `?cursor=abc&cursor=cde`
 - `?cursor[]=abc`
 - `?cursor[k1][k2]=value`
 - `?cursor[][k1][k2]=value` (?)

Long explanation here: https://evanhahn.com/gotchas-with-express-query-parsing-and-how-to-avoid-them/

Since I've found that issue, I've been using the `toString()` approach that I found somewhere, which seems the easier/safer to me. It doesn't explicitly give support to all possible corner cases that one might receive, but we probably don't want to give support to those anyway. It will work as expected if 1 value is sent or if an array is sent with only 1 element. In other cases it will generate a string that will cause an error somewhere else in the code (probably a 400, 422 or 404), but at least it won't explode a 500 due to possibly mistreating the object types. 

Anyway, if you feel like we should better handle all the `ParsedQs` possibilities let me know and I can improve that helper, maybe put it into a middleware which populates the `req` or at least in a common helper, idk. I didn't include that in this PR already as it was a bit massive already with the type changes.